### PR TITLE
fix galaxy tags

### PR DESCRIPTION
**Pull request by pavelpikta**
* update role with ansible development kit
* add molecule aws direct tests

### CODE REVIEW

File: tasks/selinux-support.yml [hunk 1]
```diff
@@ -4,17 +4,23 @@
...
   package:
-    name: "{{ selibs_item }}"
+    name: "{{ item }}"
     state: present
-  with_items:
+  register: status
+  retries: 10
+  delay: 2
+  until: status is success
+  loop:
```
Comment (tgadiev) [inline tasks/selinux-support.yml]: Package does not require loop now. Please use direct list include in module or use list variable.

File: tasks/selinux-support.yml [hunk 2]
```diff
@@ -4,17 +4,23 @@
   package:
     name: libselinux-python
     state: present
+  register: status
+  retries: 10
+  delay: 2
+  until: status is success
```
Comment (tgadiev) [inline tasks/selinux-support.yml:13]: Why to package module calls instead of one?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: tasks/selinux-support.yml (hunk 1 above)
Comment (pavelpikta) [reply tasks/selinux-support.yml]: resolve in 739e93a

File: tasks/selinux-support.yml (hunk 2 above)
Comment (pavelpikta) [reply tasks/selinux-support.yml:13]: resolve in 739e93a

**Review by tgadiev — APPROVED** [review]

**Pull request merged**